R/addins.R

```diff
@@ -0,0 +1,111 @@
+#' Style the active file
+#'
+#' Helper function for RStudio Addin.
+style_active_file <- function() {
+  file <- rstudioapi::getActiveDocumentContext()$path
```

Comment:
Do you need to call `find_active_context()` here to allow mocking?

Reply:
If we want I guess so. So far, only style_actige_region is tested but we can add a test here too to increase coverage.